content/committees/marketing/_index.md

```diff
@@ -3,4 +3,19 @@ title: "Marketing Committee"
 date: 2020-01-05T15:50:25-04:00
 ---
 
-The focus of this committee is on marketing, branding and community awareness and produces and executes the marketing plan for the release based on the Jakarta EE Key messaging provided by the Steering Committee.
\ No newline at end of file
+The focus of this committee is on marketing, branding and community awareness and produces and executes the marketing plan for the release based on the Jakarta EE Key messaging provided by the Steering Committee.
+
+<!--more-->
+
+{{< meeting-minutes-single "marketing_committee" >}}
+
+### Committee Members
+
+* Michael DeNicola - Fujitsu, Kenji Kazumura - alternate
```

Comment:
Mike has left Fujitsu. Kenji is the Fujitsu rep now

Reply:
Done! thx